src/server/accessors/MessageBuilder.ts

```diff
@@ -21,6 +21,13 @@ export class MessageBuilder implements IMessageBuilder {
         return this;
     }
 
+    public setUpdateData(data: IMessage, editor: IUser): IMessageBuilder {
+        this.msg = data;
+        this.msg.editor = editor;
+
```

Comment:
Changing the message's data would require also to set the `editedAt` field from the `IMessage` object, right? So in this case adding `this.msg.editedAt = new Date();` would accomplish that. Am I on the right path?

Reply:
There's also an `msg.updatedAt` field, shouldn't the date be assigned to this field?
I'm not sure if a message update is also an edition - maybe the date should be assigned to both fields.

Reply:
I think this is the case of checking where each of them is used. Maybe this is redundant, but if it is there, must be a reason for it. I got one instance of the usage of `editedAt` in the apps engine. Can you check this? Maybe this is used in the main repo

Reply:
Both fields are used in the main repo, so I decided to update them (with the same value).

Reply:
`editedAt` is when a message is changed by a user. `updatedAt` is whenever the system updates a message. @sampaiodiego can correct me if I'm wrong though 👀

Reply:
> `editedAt` is when a message is changed by a user. `updatedAt` is whenever the system updates a message. @sampaiodiego can correct me if I'm wrong though

that's right :+1:

Reply:
@matheusbsilva137 so we should have the `editedAt` only in this case

Reply:
Great! Only the `editedAt` field is being updated now.